src/reflect/scala/reflect/api/Trees.scala

```diff
@@ -2463,7 +2465,13 @@ trait Trees { self: Universe =>
    *  @group Traversal
    */
   class Traverser {
-    protected[scala] var currentOwner: Symbol = rootMirror.RootClass
+    /** Access from multiple threads was reported by umad.
+      * That possibly could be solved by ensuring that every unit operates on it's own copy of the tree,
+      * but it would require much bigger refactorings and would be more memory consuming.
+      */
```

Comment:
Maybe a traverser captures the currentUnit, and asserts that it is local
Maybe this is a followup investigation though